core/config/locales/en.yml

```diff
@@ -488,6 +488,9 @@ en:
           attributes:
             payment_source:
               has_to_be_payment_source_class: has to be a Spree::PaymentSource
+              not_owned_by_user: does not belong to user
+            user_id:
+              payment_source_already_exists: already has the Spree::PaymentSource in their wallet
```

Comment:
This is not a very human friendly error message. Could we use

"already has this payment source in their wallet"?

Reply:
I'm assuming @ericsaupe took inspiration from the message above. I think we could fix both in this PR.